requirements.txt

```diff
@@ -2,8 +2,7 @@ certifi==2020.4.5.1
 cycler==0.10.0
 kiwisolver==1.2.0
 matplotlib==3.1.3
-numpy==1.15.2; python_version < '3.8'
-numpy==1.17.2; python_version >= '3.8'
+numpy==1.17.2
```

Comment:
Why did you make this change? The removed lines are python version conditional statements
[https://stackoverflow.com/a/33451105](https://stackoverflow.com/a/33451105)
Removing this will change the behavior of installation.

Reply:
It seems this is changed by Jess for some unknown reasons.

Reply:
Sorry, I forgot that I changed that.  Something was breaking with my older Python 3.7 and 1.15, and therefore was breaking my older version of python 3.7.  Let me take a second look at this and see if I can figure out what is actually breaking.

Reply:
Ok, I'll hold off on approving this merge until you decide what is the better course of action on this.